src/components/atoms/BackButton/BackButton.scss

```diff
@@ -0,0 +1,35 @@
+@import "scss/constants.scss";
+
+.BackButton {
+  position: fixed;
+  top: $navbar-height;
+  z-index: z(navbar-map-back-button);
+  left: 0px;
+  height: 40px;
+  width: auto;
+  cursor: pointer;
+  background-color: $secondary;
+  box-shadow: 0 10px 20px rgba($secondary, 0.2);
+  backdrop-filter: blur(8px);
+
+  border-radius: 0 0 16px 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0 10px 0 0;
+  font-size: 0.9rem;
+
+  &:hover {
+    background-color: $lighter-intermediate-grey;
+    .back-icon {
```

Comment:
This `.back-icon` seems wrong? Doesn't align with BEM?

Reply:
It's actually no longer needed. I will remove it.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1492/commits/516bcef90616e538cbf0a8609a96d134bfbf4fcc